Fix Salt / UITK text property compatibility (#2679)
jpmorganchase/salt-ds: .changeset/orange-bulldogs-glow.md

---
"@salt-ds/core": patch
---

> Fixed components text properties impacted by global styles

Reviewer: did we want to say 'external global styles'? we also have our own global text styles in global.css (applied to the .salt-theme class) which should technically be correct for the below components so it might seem like thats not the case

Author: Updated with more details

---
"@salt-ds/core": patch
---

> Fixed component text properties (`font-weight`, `font-family`, `font-size`, `line-height`) incorrectly impacted by external global styles, which should follow text characteristic from Salt theme.

Reviewer 2: ```suggestion
Fixed component text properties (`font-weight`, `font-family`, `font-size`, `line-height`) incorrectly inheriting external global styles, which should follow the text characteristic from the Salt theme.
```
